src/formatters/__tests__/sortResults.test.ts

```diff
@@ -0,0 +1,201 @@
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 1 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 01',
+    path: ['a', 'b', 'c', 'd'],
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 2 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 02',
+    path: ['a', 'b', 'c', 'd'],
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 2 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 02',
+    path: ['a', 'b', 'c', 'e'],
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 2 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 02',
+    path: ['a', 'b', 'c', 'f'],
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 2 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 03',
+    path: ['a', 'b', 'c', 'f'],
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 2 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 03',
+    path: ['a', 'b', 'c', 'f'],
+    source: 'source 02',
+    range: {
+      start: { line: 2, character: 3 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 03',
+    path: ['a', 'b', 'c', 'f'],
+    source: 'source 02',
+    range: {
+      start: { line: 3, character: 3 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+  {
+    code: 'code 03',
+    path: ['a', 'b', 'c', 'f'],
+    source: 'source 03',
+    range: {
+      start: { line: 3, character: 3 },
+      end: { line: 99, character: 99 },
+    },
+    message: '99',
+    severity: DiagnosticSeverity.Error, // or any other level, it's irrelevant
+  },
+];
+
+describe('sortResults', () => {
+  test('should properly order results', () => {
+    let shuffled: IRuleResult[];
+    let temp: IRuleResult[];
+
+    shuffled = [...results];
+
+    do {
+      temp = [...shuffled];
+
+      shuffled = temp.sort((_a, _b) => {
+        return Math.random() * 10 - 5;
+      });
+    } while (shuffled[0].code === results[0].code);
```

Comment:
Is this loop actually needed?

Reply:
No longer needed now that the test is deterministic.